Makefile

```diff
@@ -1,18 +1,26 @@
-TOP := $(dir $(firstword $(MAKEFILE_LIST)))
+# Set our default to print out help if user has not provided a target
+.DEFAULT_GOAL := help
 
-TESTSYS_BUILD_HOST_UNAME_ARCH=$(shell uname -m)
-TESTSYS_BUILD_HOST_GOARCH ?= $(lastword $(subst :, ,$(filter $(TESTSYS_BUILD_HOST_UNAME_ARCH):%,x86_64:amd64 aarch64:arm64)))
-TESTSYS_BUILD_HOST_PLATFORM=$(shell uname | tr '[:upper:]' '[:lower:]')
-# On some hosts we get an x509 certificate error and need to set GOPROXY to "direct"
-TESTSYS_BUILD_GOPROXY ?= direct
+# Determine our root directory path for later use
+TOP := $(dir $(firstword $(MAKEFILE_LIST)))
 
+# Variables we update as newer versions are released
 BOTTLEROCKET_SDK_VERSION = v0.26.0
 BOTTLEROCKET_SDK_ARCH = $(TESTSYS_BUILD_HOST_UNAME_ARCH)
 BOTTLEROCKET_TOOLS_VERSION ?= v0.1.0
 
 BUILDER_IMAGE = public.ecr.aws/bottlerocket/bottlerocket-sdk-$(BOTTLEROCKET_SDK_ARCH):$(BOTTLEROCKET_SDK_VERSION)
 TOOLS_IMAGE ?= public.ecr.aws/bottlerocket-test-system/bottlerocket-test-tools:$(BOTTLEROCKET_TOOLS_VERSION)
 
+# Capture information about the build host
+TESTSYS_BUILD_HOST_UNAME_ARCH=$(shell uname -m)
+TESTSYS_BUILD_HOST_GOARCH ?= $(lastword $(subst :, ,$(filter $(TESTSYS_BUILD_HOST_UNAME_ARCH):%,x86_64:amd64 aarch64:arm64)))
+TESTSYS_BUILD_HOST_PLATFORM=$(shell uname | tr '[:upper:]' '[:lower:]')
+# On some hosts we get an x509 certificate error and need to set GOPROXY to "direct"
+TESTSYS_BUILD_GOPROXY ?= direct
+
+# The set of images to create. Add new artifacts here when added to the
```

Comment:
Maybe note that these are for bottlerocket images and the example images have their own targets.